### ISSUE TITLE
Committee report scraper timing out; maybe it simply doesn't exit?

### ISSUE THREAD

**Issue by aih** [aih/BillMap#573]
```
2021-09-08 06:30:00,060: ERROR/MainProcess] Task handler raised error: TimeLimitExceeded(12000.0)
Traceback (most recent call last):
  File "/home/ubuntu/.pyenv/versions/3.8.3/envs/flatgov/lib/python3.8/site-packages/billiard/pool.py", line 684, in on_hard_timeout
    raise TimeLimitExceeded(job._timeout)
billiard.exceptions.TimeLimitExceeded: TimeLimitExceeded(12000.0,)
[2021-09-08 06:30:00,060: ERROR/MainProcess] Hard time limit (12000.0s) exceeded for bills.tasks.committee_report_scrapy_task[c8aa4014-0b28-47ef-86b8-289c9566f5bf]
[2021-09-08 06:30:00,214: ERROR/MainProcess] Process 'ForkPoolWorker-4' pid:4123 exited with 'signal 9 (SIGKILL)'
```

**Comment by aih** [comment]
```
'start_time': datetime.datetime(2021, 9, 8, 3, 10, 0, 238266)}
[2021-09-08 03:10:00,380: INFO/ForkPoolWorker-4] Spider closed (finished)
[2021-09-08 03:10:00,380: WARNING/ForkPoolWorker-4] 2021-09-08 03:10:00 [scrapy.core.engine] INFO: Spider closed (finished)
[2021-09-08 06:30:00,061: INFO/ForkPoolWorker-4] Received SIGTERM, shutting down gracefully. Send again to force
```
Looks like it needs to return

**Comment by aih** [comment]
Should be fixed in 91a0522
